### PR TITLE
Documentation for groups and locations

### PR DESCRIPTION
**Pull request by AnastasiaBlack**
- Added documentation;
- Changed url name in controller.

### CODE REVIEW

File: doc/LOCATIONS.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+**Error Response :** Code: 401 Unauthorized
+
+## Show all the groups, that belong to the specified location by location Id.
+
+**URL :** /locations/{locationIds}/groups
+
+**Method :** GET
+
+**URL Params :** Required: `{locationIds}`(type: integer), example: `2`
```
Comment (svoronin) [inline doc/LOCATIONS.md]: Will it be any better if we also construct full url in example? Like this: "/locations/2/groups"

**Comment by AnastasiaBlack** [comment]
Edited an example

File: doc/GROUPS.md [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+**Success Response:** Code: 200, Content: 
+
+    [
+        {
+            "id": 1,
+            "name": "DP-115",
+            "teacher": {
+                "id": 1,
+                "firstName": "Oleg",
+                "lastName": "Shcets",
```
Comment (svoronin) [inline doc/GROUPS.md]: Can you please fix teacher's lastname in this example please?

File: doc/GROUPS.md [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+**Success Response:** Code: 200, Content:
+
+    [
+        {
+            "id": 1,
+            "name": "DP-115",
+            "teacher": {
+                "id": 1,
+                "firstName": "Oleg",
+                "lastName": "Shcets",
```
Comment (svoronin) [inline doc/GROUPS.md]: And here please

**Comment by khdimon** [comment]
Please add links in doc/README

**Comment by AnastasiaBlack** [comment]
Done!

**Pull request merged**